### PR TITLE
Even more memory to CI JDK 8

### PR DESCRIPTION
**Pull request by johanandren**

### CODE REVIEW

**Review by ennru — APPROVED** [review]
Bump it!

File: akka-http-core/src/main/scala/akka/http/ParsingErrorHandler.scala [hunk 1]
```diff
@@ -9,6 +9,8 @@ import akka.http.javadsl.{ model => jm }
 import akka.http.scaladsl.model.{ ErrorInfo, HttpResponse, StatusCode }
 import akka.http.scaladsl.settings.ServerSettings
 
+// FIXME trigger PR validation
```
Comment (patriknw) [inline akka-http-core/src/main/scala/akka/http/ParsingErrorHandler.scala:12]: @ennru accidental commit?

Comment (johanandren) [reply akka-http-core/src/main/scala/akka/http/ParsingErrorHandler.scala:12]: Removed in #4201

**Comment by johanandren** [comment]
I was thinking it is JDK 8 ergonomics changing between JDK 8 versions, making it (sometimes) guess bad limits in a container in the version we switched to. I don't have much evidence for that though. 

Note that it was OOM:ing something like 40-50% of the test runs on JDK 8, not consistently.

**Pull request merged**